src/ui/buttons/props.js

```diff
@@ -664,9 +665,6 @@ export function normalizeButtonStyle(
     tagline = false;
   }
 
-  // if color is a falsy value, set it to the default color from the funding config
```

Comment:
only concern is that there's no test change required when there's a comment saying "we need this." sounds like we're just missing a test for this entirely though?

if so, we may want to add it, since it looks like this color logic is happening across two files

Reply:
sounds good

I updated the tests, but they don't actually hit this specific use case. I tried adding `displayOnly: ['vaulatable']` and the skd would throw an error saying no valid funding sources were available.

I did add a comment in the buton.jsx file about the logic for choosing the default button color

Reply:
Looks good. I wonder if this issue could have been caught by unit tests. We should improve unit tests  with visual regressions in this module.

Reply:
The added tests at least account for taking away this comment.

If we can test it with VRT, we should be able to catch it with unit tests, but that's something we should create a ticket for in the new tech debt epic.

Reply:
it definitely could have been caught with these tests but would require some updates to the integration test server to be able to render the use case. I could have spent more time trying to make that happen, but was also unsure of how the change would affect the other tests. 

I have a lot of thought on how to improve our testing suits and will add this use case as something we should be able to better capture.